src/components/UserInfo/UserInfo.jsx

```diff
@@ -1,3 +1,18 @@
-export const UserInfo = () => (
-  <>Put the user here</>
-);
+import './UserInfo.scss';
+
+export const UserInfo = ({ user }) => {
+  const {
+    name,
+    email,
+  } = user;
+
+  return (
+    <p>
+      {' Posted by  '}
```

Comment:
Curly brackets should not be used for a regular string.

Reply:
Curly brackets are needed to add spaces around the string in this case.